public/client.js

```diff
@@ -80,7 +80,6 @@ const appendUserInfo = (room_name, description) => {
     $('.card-columns').append($userInfo);
 
 }
-
```

Comment:
remove this unnecessary change.

Reply:
That's just a line added without any code, anyways i'll revert it back !  😀